server/services/config/crm7DisplayConfig.json

```diff
@@ -590,13 +590,11 @@
       "title": "Office Use Only",
       "subsections": [
         {
-          "title": "Quality Control",
-          "fields": [
-            {
-              "label": "Decision",
-              "apiField": "decisionOfficeUseOnly"
-            }
-          ]
+          "title": "Office Use Only",
+          "customDisplay": {
+            "name": "crm7OfficeUseOnly",
+            "apiField": "OfficeUseOnly"
```

Comment:
CRM 7 API is returning OfficeUseOnly with lowercase. I have asked a ticket can be raised to change this.